src/lib/entities/product.py

```diff
@@ -10,9 +10,11 @@ def __init__(self):
         self.description = None  # string
         self.entity_status = None  # enum
         self.create_date = None  # date
+        self.delete_date = None  # date
```

Comment:
Could you create `AbstractEntity` that would contain:
1. id
2. entity_status
3. created_date
5. created_by
6. updated_date
7. updated_by

Finally make this class to be inherited by product and other entities

Reply:
done.